validate/validate.go

```diff
@@ -384,6 +384,18 @@ func (v *Validator) CheckLinux() (msgs []string) {
 		msgs = append(msgs, "rootfsPropagation must be empty or one of \"private|rprivate|slave|rslave|shared|rshared\"")
 	}
 
+	for _, maskedPath := range v.spec.Linux.MaskedPaths {
+		if !filepath.IsAbs(maskedPath) {
```

Comment:
Nothing to do with a Windows *runtime host*, but you should be able to do host-agnostic validation with `oci-runtime-tool validate …` regardless of whether the config is for Linux/Solaris/Windows or your validating host is Linux/Solaris/Windows.

Consider the use-case where someone is running a config registry and they want to validate all the configs they host against the spec.  Maybe their registry is on a Linux box, or maybe it's on a Windows box, but either way, they should still be able to check for “If this is a Linux config, are all the paths in `linux.maskedPath` absolute?”.